lib/kino_maplibre/map_cell.ex

```diff
@@ -11,6 +11,9 @@ defmodule KinoMapLibre.MapCell do
   @geometries [Geo.Point, Geo.LineString, Geo.Polygon, Geo.GeometryCollection]
   @styles %{"street (non-commercial)" => :street, "terrain (non-commercial)" => :terrain}
 
+  @query_source %{columns: nil, type: "query", variable: "Geocoding source"}
```

Comment:
I think we don't need "source" on the name. I was also wondering if we could annotate this especially, so it is clear to users it is not a variable. WDYT about:

```suggestion
  @query_source %{columns: nil, type: "query", variable: "🌎 Geocoding"}
```